test/fixtures/_command_offset/completer

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+case "${2-}" in
+    b|ba|bar)
+        echo bar
+        ;;
+    f|fo|foo)
+        echo foo
+        ;;
+    l)
+        echo line\\
+        echo two
+        echo long
+        ;;
+    li*)
+        echo line\\
+        echo two
+        ;;
+    lo*)
+        echo long
+        ;;
```

Comment:
I'd like to add the corner case where the last line is terminated by `\`. What do you think?

```suggestion
        ;;
    cont1*)
        echo cont10
        echo cont11\\
        ;;
```

Reply:
Assuming that's how bash handles it, sure.

Reply:
Assuming that's how bash handles it, sure.